src/controller.cpp

```diff
@@ -623,13 +641,14 @@ void Controller::statusUpdated(const QString& serverIpv4Gateway,
 }
 
 QList<IPAddressRange> Controller::getAllowedIPAddressRanges(
-    const Server& server) {
+    const QList<Server>& serverList) {
   logger.debug() << "Computing the allowed ip addresses";
 
   bool ipv6Enabled = SettingsHolder::instance()->ipv6Enabled();
 
   QList<IPAddress> excludeIPv4s;
   QList<IPAddress> excludeIPv6s;
+  const Server& server = serverList.last();
```

Comment:
Ok so, the first one is the exit node and the last one is the ingress node.

Reply:
That is correct, I can add a comment to help explain this better.